docs/configure-archive-logs.md

```diff
@@ -59,3 +60,14 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Why doesn't Argo Workflows recommend using this feature?
+
+Argo Workflows log storage facilities are quite basic. It is recommended that you use a combination of:
+
+* A fully featured kubernetes capable logging facility which will provide you with facilities for indexing, searching and managing of log storage.
+* Use [links](links.md) to connect from the Argo Workflows user interface to your logging facility
```

Comment:
This sentence shouldn't start with "Use" otherwise it doesn't make sense after "use a combination of..."

Reply:
Changed per [below comment](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1473500337)